packages/perseus/src/widgets/label-image/label-image.tsx

```diff
@@ -317,7 +320,8 @@ export class LabelImage
         return {markers};
     }
 
-    showRationalesForCurrentlySelectedChoices(rubric: LabelImageProps) {
+    // TODO(LEMS-2440): Can possibly remove rubric parameter during 2440? Or refactor
```

Comment:
Also kind of related to: https://khanacademy.atlassian.net/browse/LEMS-2519?focusedCommentId=337185

Here's a ticket specifically for this method: https://khanacademy.atlassian.net/browse/LEMS-2544

Reply:
Updated the comment to reference LEMS-2544.